plugins/linters/kube-linter/fixtures/__snapshots__/basic_v0.7.1.shot

```diff
@@ -0,0 +1,101 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`linter=kube-linter fixture=basic version=0.7.1 1`] = `
+{
+  "issues": [
+    {
+      "category": "CATEGORY_LINT",
+      "documentationUrl": "https://docs.kubelinter.io/#/generated/templates?id=latest-tag",
+      "level": "LEVEL_MEDIUM",
+      "location": {
+        "path": "basic.in.yaml",
+        "range": {
+          "endColumn": 1,
+          "endLine": 1,
+          "startColumn": 1,
+          "startLine": 1,
+        },
+      },
+      "message": "The container "sec-ctx-demo" is using an invalid container image, "busybox". Please use images that are not blocked by the \`BlockList\` criteria : [".*:(latest)$" "^[^:]*$" "(.*/[^:]+)$"]
+object: <no namespace>/security-context-demo /v1, Kind=Pod",
+      "mode": "MODE_BLOCK",
+      "ruleKey": "latest-tag",
+      "snippet": "apiVersion: v1",
+      "snippetWithContext": "apiVersion: v1
+kind: Pod
+metadata:
+  name: security-context-demo
+spec:
+  securityContext:
+    runAsUser: 1000
+    runAsGroup: 3000
+    fsGroup: 2000
+  volumes:
+  - name: sec-ctx-vol",
+      "tool": "kube-linter",
+    },
+    {
+      "category": "CATEGORY_LINT",
+      "documentationUrl": "https://docs.kubelinter.io/#/generated/templates?id=read-only-root-filesystems",
+      "level": "LEVEL_MEDIUM",
+      "location": {
+        "path": "basic.in.yaml",
+        "range": {
+          "endColumn": 1,
+          "endLine": 1,
+          "startColumn": 1,
+          "startLine": 1,
+        },
+      },
+      "message": "container "sec-ctx-demo" does not have a read-only root file system
+object: <no namespace>/security-context-demo /v1, Kind=Pod",
+      "mode": "MODE_BLOCK",
+      "ruleKey": "no-read-only-root-fs",
+      "snippet": "apiVersion: v1",
+      "snippetWithContext": "apiVersion: v1
+kind: Pod
+metadata:
+  name: security-context-demo
+spec:
+  securityContext:
+    runAsUser: 1000
+    runAsGroup: 3000
+    fsGroup: 2000
+  volumes:
+  - name: sec-ctx-vol",
+      "tool": "kube-linter",
+    },
+    {
+      "category": "CATEGORY_LINT",
```

Comment:
@noahd1 is there any information related to category of the issues in the SARIF output by the analyzer?

If so, we should consider how we might use it.

If there's no data about issue category then we have a bit of a decision... we could leave them as Lint which is effectively the default. Or, for some analyzers which are primarily security related, we override the category to Security. This may be a plugin where we want to do that as well.

Reply:
There's nothing in the sarif output to indicate category (or severity). Is this override something that can be done in the plugin definition? Or does it require CLI code?

Reply:
Oh I see how `checkov` defines this:

```toml
output_category = "vulnerability"
output_level = "medium"
```

Reply:
yes exactly